src/game/default/g_types.h

```diff
@@ -482,8 +477,6 @@ typedef enum {
  * @brief Item types.
  */
 typedef enum {
-	ITEM_NONE,
-
```

Comment:
So that uninitialized members don't appear to be real things. But if there are no side effects from this, I'm okay with that.